pkgs/misc/cups/drivers/cups-pdf-to-pdf/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cups
+, coreutils
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cups-pdf-to-pdf";
+  version = "unstable-2021-12-22";
+
+  src = fetchFromGitHub {
+    owner = "alexivkin";
+    repo = "CUPS-PDF-to-PDF";
+    rev = "c14428c2ca8e95371daad7db6d11c84046b1a2d4";
+    hash = "sha256-pa4PFf8OAFSra0hSazmKUfbMYL/cVWvYA1lBf7c7jmY=";
+  };
+
+  buildInputs = [ cups ];
+
+  postPatch = ''
+    sed -r 's|(gscall, size, ")cp |\1${coreutils}/bin/cp |' cups-pdf.c -i
+  '';
+
+  # gcc command line is taken from original cups-pdf's README file
+  # https://fossies.org/linux/cups-pdf/README
+  buildPhase = ''
+    runHook preBuild
+    gcc -O9 -s cups-pdf.c -o cups-pdf -lcups
```

Comment:
This fails on Darwin, which is definitely something that can be used there I think.

Reply:
Hmmmm, I don't know much about Darwin.  Following the first hint in https://nixos.org/manual/nixpkgs/stable/#sec-darwin , I replaced `gcc` with `$CC`.  Hopefully, this calls `clang` and the command line still works.  On `x86_64-linux`, the package still builds fine with this change (and the test passes).

Can you try if this works on Darwin.

Reply:
According to OfBorg, this works on Darwin: https://logs.nix.ci/?key=nixos/nixpkgs.182360&attempt_id=e17af287-4a25-4e81-bc11-09cf5d8e3f13